### ISSUE TITLE
Website is not responsive

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Residential-Building-Brokers#10]
![image](https://cloud.githubusercontent.com/assets/12791909/11432768/c8542e08-94ef-11e5-96cc-ddf4b5ed311d.png)

![image](https://cloud.githubusercontent.com/assets/12791909/11432773/f1706b76-94ef-11e5-8de4-0f3b4884226a.png)

**Comment by titanguest** [comment]
This is per Bernard's request, as it is need to be raised to Jolene for confirmation.

**Comment by ermatitan** [comment]
A confirmation is needed before we can close the issue. Thanks

**Comment by ermatitan** [comment]
Won't fix issue as confirmed by Bernard Salas
